### PR TITLE
Addition of Watteco Vaqao and Vaqao+ 

### PR DESCRIPTION
**Pull request by franckLG**
Added codecs for the 2 sensors
Simplified the configuration files and overall approach so to be replicable for all watteco devices. For most cases, only const declarations in uplink_decoder.js would be needed

### CODE REVIEW

File: manufacturers/Watteco/models/Vaqao/uplink_decoder.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    var time = process.argv[4];
+    // ********* Test pattern (uncomment to test behaviour) ********************
+        // Pattern batch
+        // payload = "424500000111008FC85EC128B4872088AD060442001E3322118A98782B";
+        // Pattern "uplink standard report containing firmware version"
+        // payload = "110100000002000D0305020015E2"
+        // Pattern "uplink standard report containing humidity value"
+        // payload = "110A04050000210E89"
+        // Pattern "uplink standard report containing dataup info" (IGNORED) 
+        payload = "110180040000000800"
```
Comment (LeaRobertEGM) [inline manufacturers/Watteco/models/Vaqao/uplink_decoder.js:28]: This should be commented, right ? And I tried to execute the script with one of the test pattern but nothing was returned.

File: manufacturers/Watteco/models/Vaqao/uplink_decoder.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    humidity: ["humidity_1","humidity_2"],
+    pin_state:["violation_detection"]
+}
+
+function main() {
+    var payload = process.argv[3];
+    var time = process.argv[4];
+    // ********* Test pattern (uncomment to test behaviour) ********************
+        // Pattern batch
+        // payload = "424500000111008FC85EC128B4872088AD060442001E3322118A98782B";
```
Comment (gasserluc) [inline manufacturers/Watteco/models/Vaqao/uplink_decoder.js:22]: When trying with this payload, I get 
```
{"temperature_1":[{"type":"Property","value":26.3,"observedAt":"2024-06-13T07:41:12.976Z","unitCode":"CEL","datasetId":"urn:ngsi-ld:Dataset:Raw"},{"type":"Property","value":26.3,"observedAt":"2024-06-13T07:41:17.976Z","unitCode"
:"CEL","datasetId":"urn:ngsi-ld:Dataset:Raw"}],"humidity_1":[{"type":"Property","value":41,"observedAt":"2024-06-13T07:41:12.976Z","unitCode":"P1","datasetId":"urn:ngsi-ld:Dataset:Raw"},{"type":"Property","value":41,"observedAt"
:"2024-06-13T07:41:17.976Z","unitCode":"P1","datasetId":"urn:ngsi-ld:Dataset:Raw"}],"CO2":[{"type":"Property","value":1540,"observedAt":"2024-06-13T07:41:14.976Z","unitCode":"52","datasetId":"urn:ngsi-ld:Dataset:Raw"},{"type":"P
roperty","value":1500,"observedAt":"2024-06-13T07:41:19.976Z","unitCode":"52","datasetId":"urn:ngsi-ld:Dataset:Raw"}],"TVOC":[{"type":"Property","value":30,"observedAt":"2024-06-13T07:41:12.976Z","datasetId":"urn:ngsi-ld:Dataset
:Raw"},{"type":"Property","value":30,"observedAt":"2024-06-13T07:41:17.976Z","datasetId":"urn:ngsi-ld:Dataset:Raw"}]}
```
We can't have "temperature_1" in the entity, it needs to be "temperature"

Comment (franckLG) [reply manufacturers/Watteco/models/Vaqao/uplink_decoder.js:22]: Should it ? Some of the Watteco sensors will return temperature_1 and temperature_2. How to handle it ? The 2 options I saw are 1. keeping this as a property name or 2. use of dataset.
I choose option 1 as mapping toward a temperature property would be made when connecting the Device to the observedEntity. 
Option 2 should be feasible as well.

File: manufacturers/Watteco/models/Vaqao/uplink_decoder.js (hunk 1 above)
Comment (franckLG) [reply manufacturers/Watteco/models/Vaqao/uplink_decoder.js:28]: Yes, indeed

File: manufacturers/Watteco/models/Vaqao/uplink_decoder.js (hunk 2 above)
Comment (gasserluc) [reply manufacturers/Watteco/models/Vaqao/uplink_decoder.js:22]: I don't really like option 1 (very basic example: if you do a query to get all Device that measures temperature, you won't get this one) but it is true that if we use 2 datasetId and try to map them to the same observedEntity, they will be mapped to the same datasetId on the observedEntity (urn:ngsi-ld:Dataset:Device:*devEui*:Raw). 
I still think we should go for option 2, we just need to do a simple evolution of the mapping module for it to work : 
if the source datasetId is urn:ngsi-ld:Dataset:*whatever*:Raw, the target datasetId would be urn:ngsi-ld:Dataset:Device:*devEui*:*whatever*:Raw (in this case *whatever* would be "Temperature1", "1" or "Probe1", smth like that)

Comment (franckLG) [reply manufacturers/Watteco/models/Vaqao/uplink_decoder.js:22]: OK. 
For the time being I did not fund any mention of temperature_2 (or humidity_2) in the devices specifications (http://support.nke-watteco.com/wp-content/uploads/2017/02/LoRaWAN_Sensors_Application_Layer_Description.pdf) and codecs examples so they seem unused. So I simply changed temperature_1 to temperature

File: manufacturers/Watteco/models/Vaqao/uplink_decoder.js (hunk 1 above)
Comment (franckLG) [reply manufacturers/Watteco/models/Vaqao/uplink_decoder.js]: corrected in last update

**Comment by gasserluc** [comment]
@franckLG I have replaced "CO2" and "TVOC" by "co2" and "tvoc" as it is the convention in NGSI-LD (see for example: https://swagger.lab.fiware.org/?url=https://smart-data-models.github.io/dataModel.Environment/AirQualityObserved/swagger.yaml).

**Review by gasserluc — APPROVED** [review]

**Pull request merged**